src/react/components/pages/editorPage/editorPage.test.tsx

```diff
@@ -112,26 +134,19 @@ describe("Editor Page Component", () => {
         });
 
         const wrapper = createComponent(store, props);
-        const editorPage = wrapper.find(EditorPage).childAt(0) as ReactWrapper<IEditorPageProps, IEditorPageState>;
```

Comment:
why doesn't this have to typed anymore?

Reply:
Sonar was complaining about an unnecessary cast